docs/docsource/operation/XDM_GUI/create_an_account.rst

```diff
@@ -0,0 +1,35 @@
+.. _Create an Account:
+
+Create an Account
+=================
+
+Because |product| cannot do anything without at least one account to work with,
+the first step in any working deployment it to create an account. To create an
+account:
+
+#. Log into the XDM UI.
+
+   .. image:: ../Graphics/xdm_ui_login.png
+
+#. The **CREATE NEW ACCOUNT** window displays.
+
+   .. image:: ../Graphics/xdm_ui_create_account.png
+
+   Enter a unique account name and the email address for the account
+   administrator (root). You can also use the **Quota in GB** field to set a
+   hard upper limit on space available to this account. This limit can be raised
```

Comment:
"This limit can be raised": it is not the case right now. @terrydervaux

Reply:
@nicolas2bert Is this limitation from UI side or Vault does not support it as of now.
If its Vault let me know, I will check the call to update the account quota in Vault

Reply:
Vault supports [update account quota](https://github.com/scality/Vault/blob/development/8.2/PROTOCOL.md#update-account-quota).
Pensieve-API as well as Zenko-UI do not.

Reply:
sounds good. Just wanted to make sure you are not facing issues with Vault :)

Reply:
Do we want to mention the Vault API option to customers, or is this something we want to keep "under the hood" so that we don't have to support/clean up after them mucking about with Vault API calls?

Reply:
Reviewing the documentation, "quota" is nowhere to be seen in the existing Zenko docs, including APIs. This indicates either incomplete documentation or a positive decision to elide Valut API details from this platform. @rahulreddy: do you have any recollection of this as a UX decision?

Reply:
I don't think we should mention the create account Vault API. 
Users do not have access to Vault Admin credentials and so will not be able to create accounts directly through Vault API. Let's keep it under the hood for now.

Reply:
It's ok  for a first iteration to keep it "under the hood" since we don't have the documentation to do it programmatically -> created [ZENKO-3040](https://scality.atlassian.net/browse/ZENKO-3040).

Eventually, we'll be able to update the quota in the UI -> created [ZENKO-3039](https://scality.atlassian.net/browse/ZENKO-3039) w/ your comments.